### ISSUE TITLE
Make the script less spammy

### ISSUE THREAD

**Issue by philbooth** [mozilla/fxa-sms-forecast#1]
From @irrationalagent, via email:

> A few things off the top of my head:
>
> 1. Run the model only once or at most twice a day. We're trying to predict 7days out so waiting a day shouldn't change things that much. When the model gets it wrong we won't get spammed as much
>
> 2. The model defines a cycle as 24 hours, the more cycles we have the better it will perform. Waiting 7 days before doing the prediction might be a good idea. Seems very unlikely that we will need it during those points.
>
> 3. The upper and lower bounds are 95% confidence intervals, so using the upper bound means we are using the very edge of the predicted range. Changing 95 to some other number is trivial. We could turn it to 75 for example. This will narrow the interval. The interval gets stupidly wide with lots of hour to hour variability and with less data. The negative numbers are side effect.
>
> 4. The actual predictions are run on the non cumulative spends per hour, and then the prediction of the total spend is generated by adding back in the cumulative totals. To make the model more accurate at the beginning of the month we can continue the predictions forward from the previous month but reset the total to 0 when the hour crosses over into the next month. 
>
> 5. I could make a better model that actually takes into account events, day of week, holidays, objective date (for seasonality), etc. Would just take time and a lot of hand tuning.

**Comment by philbooth** [comment]
@jbuck, re: point 1 above, let's change the SMS script to run once per day.

**Comment by philbooth** [comment]
> Waiting 7 days before doing the prediction might be a good idea.

We already do this one fwiw (we also stop forecasting 7 days out from the end of the month). We could increase that period to more than 7 days though.

**Comment by philbooth** [comment]
> The upper and lower bounds are 95% confidence intervals, so using the upper bound means we are using the very edge of the predicted range. Changing 95 to some other number is trivial. We could turn it to 75 for example.

I like the sound of this one, will make the change.

**Comment by irrationalagent** [comment]
good to know. i think doing (4) could help us here. shouldn't be too hard but it would require changing the aws commands to return more data. I'll fool around with it when I get a spare few

**Comment by jbuck** [comment]
I've just put up a PR to run it daily

**Comment by irrationalagent** [comment]
in the email, we could also be more specific about what the numbers mean, something like:

```
Lower end of forecast spend, 7 days from now: XXXXX
Average of forecast spend, 7 days from now: XXXXX
Upper end of forecast spend, 7 days from now: XXXXX

The forecasting model assigns a 75% chance that the total spend 7 days from now will be between the lower and upper ends shown above.
```

**Comment by irrationalagent** [comment]
it might also be good to include where the spend currently is, to give context

**Comment by philbooth** [comment]
@irrationalagent, should we close this now or do we still want to do more stuff?

**Comment by irrationalagent** [comment]
closing for now, if i ever get around to training a better model ill open a new issue
